src/fluree/db/api/query.cljc

```diff
@@ -28,42 +26,7 @@
 ;; main query interface for APIs, etc.
 
 
-(declare query query-async)
-
-
-(defn query-type
```

Comment:
I'm a fan of removing this dispatch and having each query type just be called directly. Does this break any expectations of how to use the query API (ie does this require any documentation changes or anything)?

Reply:
Good catch; it does. I added [a `multi-query` api function](https://github.com/fluree/db/pull/358/files#diff-8252fe8c552b17d3d9a98e8e24e7bb197a83dc5081981389a0803626f4cef56eR270-R274) that might need documentation.

I hope to eventually roll all the multi-query functionality into the normal `fql/query` process, but I got tripped up dealing with the different options maps when I tried to unify everything.